src/lib/AnnotationControls.js

```diff
@@ -0,0 +1,63 @@
+import noop from 'lodash/noop';
+import { ICON_REGION_COMMENT } from './icons/icons';
+import Controls, { CLASS_BOX_CONTROLS_GROUP_BUTTON } from './Controls';
+
+const CLASS_ANNOTATIONS_GROUP = 'bp-annotations-group';
+const CLASS_REGION_COMMENT_BUTTON = 'bp-region-comment-btn';
+const CLASS_BUTTON_ACTIVE = 'active';
+
+class AnnotationControls {
+    /** @property {Controls} - Controls object */
+    controls;
+
+    /** @property {Boolean} - Region comment mode active state */
+    isRegionCommentActive = false;
```

Comment:
We've been referring to these as "area highlights" or just area annotations. May want to check with PM on official naming.

Reply:
> We've been referring to these as "area highlights" or just area annotations. May want to check with PM on official naming.

I'm confusing too, because the design is using "region comment" and tooltip hint is "Comment on Region".